web-app/packages/lib/src/modules/project/components/ProjectsTable.vue

```diff
@@ -55,7 +55,7 @@ SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
               ><PTag
                 v-if="slotProps.data.access.public && !onlyPublic"
                 severity="success"
-                :pt="{ root: { class: 'p-1 ml-1' } }"
+                :pt="{ root: { class: 'ml-1 pl-0.5 pr-0.5' } }"
```

Comment:
I think, we can remove this classes completely and use just PTag as in "Recommended".